src/platform/web/ui/session/room/RoomView.js

```diff
@@ -57,7 +58,7 @@ export class RoomView extends TemplateView {
                         new TimelineView(timelineViewModel) :
                         new TimelineLoadingView(vm);    // vm is just needed for i18n
                 }),
-                t.view(bottomView),
+                bottomView ? t.view(bottomView) : text(''),
```

Comment:
Just trying to not render the composer when no `vm.composerViewModel.kind` matches with the goal to hide the composer.